src/update-feeds-loops/contracts.ts

```diff
@@ -3,12 +3,36 @@ import {
   AirseekerRegistry__factory as AirseekerRegistryFactory,
   Api3ServerV1__factory as Api3ServerV1Factory,
 } from '@api3/contracts';
+import { go } from '@api3/promise-utils';
 import { ethers } from 'ethers';
 import { zip } from 'lodash';
 
 import type { AirnodeAddress, TemplateId } from '../types';
 import { decodeDapiName, deriveBeaconId } from '../utils';
 
+export const createProvider = async (url: string) => {
```

Comment:
@aquarat provided a nicer alternative to this with the following gist:
```
const fetchClient = new FetchRequest(chain.rpcUrl);
  fetchClient.timeout = 10_000;

  const network = {
    name: chain.name,
    chainId: chain.chainId,
  };
  const providerPublic = new JsonRpcProvider(fetchClient, network, 
  {
    staticNetwork: Network.from(network.chainId), // or just network
  });
```

We know the chainID, but not the chain name. We could use `@api3/chains` for this, but I quite like that we "detect" the faulty providers this way, so I am keeping it this way. Shout if you have some strong preferences.